ward/collect.py

```diff
@@ -13,18 +13,22 @@ def is_test_module(module: pkgutil.ModuleInfo) -> bool:
     return module.name.startswith("test_")
 
 
-def get_info_for_modules(path: str) -> Generator[pkgutil.ModuleInfo, None, None]:
-    # Check for modules at the root of the specified path
-    for module in pkgutil.iter_modules([path]):
+def get_info_for_modules(path: List[str]) -> Generator[pkgutil.ModuleInfo, None, None]:
+    # If multiple paths are specified, remove duplicates
+    paths = list(set(path))
```

Comment:
This raises an interesting question: what happens if you have two paths that aren't strictly equal, but one is contained within the other? For example, imagine a user supplies the paths `./tests/a` and `./tests/a/b`. The tests inside `./tests/a/b` will be collected twice.

There's a few different ways I can think of getting around this:
1. Before entering a subdirectory to look for tests, we check if it that directory has already been supplied separately by the user, and if it has we skip over it, knowing it will be dealt with. There's probably some other edge cases in here I haven't thought of though 😄 
2. We just collect all the tests from every path, then filter out the duplicates afterwards.

Option 1 is my favourite, but let me know what you think.

Reply:
I see what you mean. I'll try to come up with a solution a bit later today

Reply:
I ended up just creating a list of checked directories and only added directories that hadn't been seen before. It seems straight forward enough that I can't imagine any complications.